ui/src/app/services/fetch/fetch.service.ts

```diff
@@ -209,13 +210,21 @@ export class FetchService {
     return this.http.post(`${this.url}/restore/tables`, payload)
   }
 
+  validateCheckConstraint() {
```

Comment:
Where is this API used?

Reply:
now this api will be replaced with expression_verification